src/app/file/file-detail.tpl.html

```diff
@@ -1,68 +1 @@
-<md-toolbar class="content-toolbar">
-  <div class="md-toolbar-tools">
-    <h3>
-      {{ctrl.file.name}}
-    </h3>
-
-    <span flex></span>
-    <!-- <md-button class="md-icon-button" aria-label="Edit" ui-sref="app.file.edit({fileId: ctrl.file.id})">
-      <md-icon md-font-set="material-icons">edit</md-icon>
-    </md-button>
-    <md-button class="md-icon-button" aria-label="Delete" ng-click="ctrl.delete($event, ctrl.file)">
-      <md-icon md-font-set="material-icons">delete</md-icon>
-    </md-button> -->
-  </div>
-</md-toolbar>
-<!-- <md-button class="md-fab" aria-label="New file" ui-sref="app.file.new">
-  <md-icon md-font-set="material-icons">add</md-icon>
-</md-button> -->
-
-<div class="wfm-maincol-scroll">
-<md-list>
-  <md-list-item class="md-2-line">
-    <md-icon md-font-set="material-icons">info</md-icon>
-    <div class="md-list-item-text">
-      <h3>{{ctrl.file.id}}</h3>
-      <p>Id</p>
-    </div>
-    <md-divider></md-divider>
-  </md-list-item>
-
-  <md-list-item class="md-2-line">
-    <md-icon md-font-set="material-icons">photo_album</md-icon>
-    <div class="md-list-item-text">
-      <h3>{{ctrl.file.name}}</h3>
-      <p>Name</p>
-    </div>
-    <md-divider></md-divider>
-  </md-list-item>
-
-  <md-list-item class="md-2-line">
-    <md-icon md-font-set="material-icons">info_outline</md-icon>
-    <div class="md-list-item-text">
-      <h3>{{ctrl.file.uid}}</h3>
-      <p>Uid</p>
-    </div>
-    <md-divider></md-divider>
-  </md-list-item>
-
-  <md-list-item class="md-2-line">
-    <md-icon md-font-set="material-icons">account_box</md-icon>
-    <div class="md-list-item-text">
-      <h3>{{ctrl.workerMap[ctrl.file.owner].name}}</h3>
-      <p>Owner</p>
-    </div>
-    <md-divider></md-divider>
-  </md-list-item>
-
-  <md-list-item class="md-2-line with-image">
-    <md-icon md-font-set="material-icons">photo</md-icon>
-    <div class="md-list-item-text">
-      <h3><img wfm-img uid="ctrl.file.uid" style="width:450px"></h3>
-    </div>
-    <md-divider></md-divider>
-  </md-list-item>
-
-</md-list>
-
-</div><!-- wfm-maincol-scroll -->
+<file-detail file="ctrl.file" display-options="ctrl.displayOptions"></file-detail>
```

Comment:
Can you add a comment here on where this directive is coming from.

This is something we should do whenever editing the mobile or portal apps. It will improve quality as we go,